tensorflow/python/keras/benchmarks/keras_examples_benchmarks/mnist_conv_benchmark_test.py

```diff
@@ -96,21 +96,16 @@ def benchmark_conv_mnist_bs_256(self):
     self.report_benchmark(
         iters=run_iters, wall_time=wall_time, metrics=metrics, extras=extras)
 
-  def benchmark_conv_mnist_bs_256_gpu_2(self):
-    """Measure performance with batch_size=256, run_iters=3, gpu=2 and
-
-    distribution_strategy='mirrored'
-    """
-    batch_size = 256
+  def benchmark_conv_mnist_bs_512(self):
+    """Measure performance with batch_size=512 and run_iters=3."""
+    batch_size = 512
     run_iters = 3
```

Comment:
Can we use a larger `run_iters`, if it doesn't take too long for one iter? like 5?